scrapy/pipelines/__init__.py

```diff
@@ -22,3 +22,21 @@ def _add_middleware(self, pipe):
 
     def process_item(self, item, spider):
         return self._process_chain('process_item', item, spider)
+
+
+class ItemPipeline(object):
+    """Base class for scrapy item pipelines.
```

Comment:
I would suggest name it `ItemPipelineBase` or  `BaseItemPipeline`. I think it makes clearer that is just a base class intended to be inherited from.